addons/hr_timesheet/models/hr_employee.py

```diff
@@ -7,6 +7,22 @@
 class HrEmployee(models.Model):
     _inherit = 'hr.employee'
 
+    def name_get(self):
+        res = super().name_get()
+        if len(self.env.context.get('allowed_company_ids', [])) <= 1:
+            return res
+        name_mapping = dict(res)
+        employee_read_group = self.env['hr.employee'].sudo()._read_group(
+            [('user_id', 'in', self.user_id.ids)],
+            ['user_id'],
+            ['user_id'],
+        )
+        employees_count_per_user = {res['user_id'][0]: res['user_id_count'] for res in employee_read_group}
```

Comment:
```suggestion
        employees_count_per_user = {
            res['user_id'][0]: res['user_id_count']
            for res in self.env['hr.employee'].sudo()._read_group(
                [('user_id', 'in', self.user_id.ids)],
                ['user_id'],
                ['user_id'],
            )
        }
```
Small detail number 1 :angel: